src/main/webapp/WEB-INF/view/bots.jsp

```diff
@@ -0,0 +1,35 @@
+<%@ page import="codeu.model.data.Bot" %>
+<%@ page import="java.util.List" %>
+<%
+List<Bot> bots = (List<Bot>) request.getAttribute("bots");
+%>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Bots</title>
+  <%@ include file="navbar.jsp" %>
+  <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+
+  <div id="container">
+    <h1>Bots
+      <a href="" style="float: right">&#8635;</a></h1>
```

Comment:
What does this accomplish?

Reply:
It was to update the bots, but I now realize this only happens when the site is updated. I'll remove it.